spec/acceptance/conf_setting_spec.rb

```diff
@@ -1,42 +1,33 @@
 require 'spec_helper_acceptance'
 
-tmpdir = default.tmpdir('tmp')
+tmpdir = setup_test_directory
 
 describe 'hocon_setting resource' do
   after :all do
-    shell("rm #{tmpdir}/*.conf", acceptable_exit_codes: [0, 1, 2])
+    run_shell("rm #{tmpdir}/*.conf", acceptable_exit_codes: [0, 1, 2])
   end
 
   shared_examples 'has_content' do |path, pp, content|
     before :all do
-      shell("rm #{path}", acceptable_exit_codes: [0, 1, 2])
-    end
-    after :all do
-      shell("cat #{path}", acceptable_exit_codes: [0, 1, 2])
-      shell("rm #{path}", acceptable_exit_codes: [0, 1, 2])
+      run_shell("rm #{path}", expect_failures: true)
     end
 
     it 'applies the manifest twice' do
-      apply_manifest(pp, catch_failures: true)
-      apply_manifest(pp, catch_changes: true)
+      idempotent_apply(pp)
     end
 
     describe file(path) do
       it { is_expected.to be_file }
       # XXX Solaris 10 doesn't support multi-line grep
-      it("should contain #{content}", unless: fact('osfamily') == 'Solaris') {
+      it("should contain #{content}") {
```

Comment:
We dropped the guard but not the comment about it, is this intentional?

Reply:
its not supported in the metadata.json. Thought it was better to rip it out